### ISSUE TITLE
Icon request: rss

### ISSUE THREAD

**Issue by tborychowski** [google/material-design-icons#178]
please add rss icon. FA and ALL icon libraries have one. I'd prefer not to include the whole icon library (in addition to material icons) just for that one...

**Comment by shyndman** [comment]
Hey @tborychowski,

That's definitely an important one. I've put in a request with our designers. If they have cycles maybe they'll help out.

**Comment by simodorg** [comment]
Yep, it is very useful in website development.

**Comment by frankros91** [comment]
I've been using the wifi icon since it's the most similar.  But I agree, an rss icon is needed.

**Comment by javmarina** [comment]
Although not created by Google itself, there is a nice one here: https://materialdesignicons.com/icon/rss-box

**Comment by thewebvy** [comment]
This would be greatly appreciated.

**Comment by Arsyady** [comment]
+1

Really needed here.
Thanks a lot

**Comment by joshribakoff** [comment]
I installed the package from bower, and I have 2.2.0 according to the package.json found inside it. I added this to my page:

```
<link rel="stylesheet" href="bower_components/material-design-icons/iconfont/material-icons.css" />
```
I'm trying to use the new RSS icon shown here:
![selection_256](https://cloud.githubusercontent.com/assets/4021306/13336686/45347518-dbcc-11e5-9566-17b4c9b0e6b1.png)

Here's how it shows up on my page (next to another icon that does work):
![selection_257](https://cloud.githubusercontent.com/assets/4021306/13336691/5331661c-dbcc-11e5-94ea-c7cbe90e2869.png)

Additionally, I do appear to have the files for this icon when I search:
![menu_258](https://cloud.githubusercontent.com/assets/4021306/13336715/7409cc6c-dbcc-11e5-9ad8-5cdd36b5e112.png)

Any idea why its not working? I'll try out the wifi icon as suggested earlier in the thread.
